src/MarkdownTextInput.web.tsx

```diff
@@ -205,6 +206,7 @@ const MarkdownTextInput = React.forwardRef<MarkdownTextInput, MarkdownTextInputP
           {whiteSpace: multiline ? 'pre-wrap' : 'nowrap'},
           disabled && styles.disabledInputStyles,
           parseToReactDOMStyle(flattenedStyle),
+          caretHidden && styles.caretHidden,
```

Comment:
Please also add `caretHidden` to the dependency array.

Reply:
Done